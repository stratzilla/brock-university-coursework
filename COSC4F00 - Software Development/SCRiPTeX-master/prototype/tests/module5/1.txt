35
55
17
14
52
30
18
12
20
37
27
24
32
23
41
24
33
45
30
22
25
28
50
31
14
27
55
17
14
47
10
35
34
39
24
16
