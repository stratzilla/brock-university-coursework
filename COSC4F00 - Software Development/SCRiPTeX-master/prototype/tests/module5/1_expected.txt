35
T
h
e
Q
u
i
c
k
B
r
o
w
n
F
o
x
J
u
m
p
s
O
v
e
r
T
h
e
L
a
z
y
D
o
g
